app/assets/js/rtMedia.js

```diff
@@ -1241,3 +1241,31 @@ function get_parameter( parameter, data ) {
 		return false;
 	}
 }
+
+/**
+ * Check if upload term is active then force user to check I agree term.
+ */
+function rtm_allow_upload_terms_activity() {
+
+	jQuery('#whats-new').click(function () {
+		var rtmedia_upload_terms_conditions = jQuery('#rtmedia_upload_terms_conditions');
+		var aw_whats_new_submit = jQuery('#aw-whats-new-submit');
+		// Check if I agree terms exist or not in activity page
+		if (rtmedia_upload_terms_conditions.length > 0) {
+			// Bydefault prevent user to post update.
+			aw_whats_new_submit.attr('disabled', 'disabled');
+			// Handle check event of I agree term.
+			rtmedia_upload_terms_conditions.change(function () {
+				if (rtmedia_upload_terms_conditions.prop('checked')) {
+					aw_whats_new_submit.removeAttr('disabled');
+				}
```

Comment:
JFI: If checkbox is checked then you remove attribute `disabled`. What if checkbox is first checked and then unchecked, have you tested the scenarios properly? @malavasita

Reply:
Yes I have checked with possible scenarios. @SID177